force-app/main/default/classes/SOQL.cls

```diff
@@ -77,6 +100,10 @@ public with sharing class SOQL {
     //[public | private ] [static] data_type method_name (input parameters) {
     //  The body of the method
     //}
+    public static Integer getAccountCount () {
+        Integer accountCount = [SELECT COUNT() FROM Account];
```

Comment:
return directly?

Reply:
In this case am I suppose to return the Querry or variable?